From: =?utf-8?q?Piotr_O=C5=BCarowski?= <ozarow@gmail.com>
Date: Sun, 6 Nov 2016 22:11:09 +0100
Subject: do not add changelog to long description

it contains non-ascii characters (and thus FTBFS) and we install it via
separate file anyway
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index d47cabc..626fc62 100644
--- a/setup.py
+++ b/setup.py
@@ -47,8 +47,8 @@ args = dict(
     name="yarl",
     version=version,
     description=("Yet another URL library"),
-    long_description="\n\n".join([read("README.rst"), read("CHANGES.rst")]),
-    long_description_content_type="text/x-rst",
+    # long_description="\n\n".join([read("README.rst"), read("CHANGES.rst")]),
+    # long_description_content_type="text/x-rst",
     classifiers=[
         "License :: OSI Approved :: Apache Software License",
         "Intended Audience :: Developers",
